src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Lights extends SubsystemBase {
+    public enum LightCode{
+        GAME_OBJECT, // green
+        READY_TO_SCORE, // blue
+        NO_TAG, // red
+        ALIGNING_TO_TAG, // yellow
+        PARTY_MODE, // rainbow 🌈
+        DEFAULT, // green back and forth (larson animation)
+        OFF
+    }
+    public Lights(){}
+    @Override
+    public void periodic(){}
+    public void setLEDs(){}
+    public void setStatusFramePeriod(){}
```

Comment:
Note: once we're more familiar with how a CANdle works, we might change these up to something that makes more sense. It's hard for me to say right now, since I've never used one before. We'll see. LEDs are lower priority.